native/python/jp_pythontypes.cpp

```diff
@@ -358,11 +358,11 @@ JPPyObjectVector::JPPyObjectVector(PyObject* inst, PyObject* sequence)
 {
 	m_Instance = JPPyObject::use(inst);
 	m_Sequence = JPPyObject::use(sequence);
-	size_t n = 0;
+	Py_ssize_t n = 0;
 	if (sequence != nullptr)
 		n = PySequence_Size(m_Sequence.get());
 	m_Contents.resize(n + 1);
-	for (size_t i = 0; i < n; ++i)
+	for (auto i = 0; i < n; ++i)
```

Comment:
This is a case where I really don't like the use of auto.   Read literally it says ``auto i = 0;``  but ``0`` has no inherent type info.   I am guessing the compiler  is supposed to guess from the comparison statement.   

I think in this case I would prefer being explicit.   It is fine to use auto when we are doing ``auto v = (Type) u;`` where the type of the variable is clear, but letting it simply guess on context from another statement seems unnecessary.

Reply:
I used decltype(n) to let the compiler use the type of the condition variable.